### PR TITLE
Small additions for light Presets

### PR DESCRIPTION
**Pull request by andrewsilke**
Adding "load text file" and "save text file" to filesystem.py 
Also added a function for finding the temp directory for each os

### CODE REVIEW

File: zoo/libs/utils/filesystem.py [hunk 1]
```diff
@@ -447,6 +447,40 @@ def loadFile(filepath):
...
+    :param pathDirectory: the directory path
+    :type pathDirectory: str
+    :param fileName: the file name
+    :type fileName: str
+    :return textFile: the text file
+    :rtype textFile: str
+    """
+    currentFile = os.path.join(pathDirectory, fileName)
+    with open(currentFile, "r") as myfile:  # add returns
+        textFile = myfile.read().replace('', '')
```
Comment (dsparrow27) [inline zoo/libs/utils/filesystem.py]: ? replace an empty string with an empty string?

File: zoo/libs/utils/filesystem.py [hunk 2]
```diff
@@ -447,6 +447,40 @@ def loadFile(filepath):
...
+
+def saveFileTxt(formattedText, newfile):
+    """Saves a string to a file
+
+    :param formattedText: the text
+    :type formattedText: str
+    :param newfile: the file to be saved full path
+    :type newfile: str
+    """
+    file = open(newfile, "wb")
```
Comment (dsparrow27) [inline zoo/libs/utils/filesystem.py]: use the with statement

File: zoo/libs/utils/filesystem.py [hunk 3]
```diff
@@ -447,6 +447,40 @@ def loadFile(filepath):
...
+    file.write(formattedText)
+    file.close()
+
+
+def osTempDirectory():
```
Comment (dsparrow27) [inline zoo/libs/utils/filesystem.py]: this should be a need function, one liners shouldn't have functions . use "tempfile" std library

File: zoo/libs/utils/filesystem.py [hunk 4]
```diff
@@ -447,6 +447,40 @@ def loadFile(filepath):
         yield f
 
 
+def loadFileTxt(pathDirectory, fileName):
```
Comment (dsparrow27) [inline zoo/libs/utils/filesystem.py]: painful set of arguments,  just have a "filePath" argument

File: zoo/libs/utils/filesystem.py (hunk 2 above)
Comment (andrewsilke) [reply zoo/libs/utils/filesystem.py]: got it

File: zoo/libs/utils/filesystem.py (hunk 3 above)
Comment (andrewsilke) [reply zoo/libs/utils/filesystem.py]: Yeah ok, I've been using a few one-liners when it's going to be difficult to remember the code, I find I'm always googling and different solutions all over the web, can get messy, but I didn't know about the tempfile standard library, sounds good, on it.

File: zoo/libs/utils/filesystem.py (hunk 4 above)
Comment (andrewsilke) [reply zoo/libs/utils/filesystem.py]: sure

File: zoo/libs/utils/filesystem.py (hunk 3 above)
Comment (andrewsilke) [reply zoo/libs/utils/filesystem.py]: yeah fab is just 
tempfile.gettempdir()
noice

**Comment by andrewsilke** [comment]
Yep this is all fixed good to go

File: zoo/libs/utils/filesystem.py (hunk 3 above)
Comment (dsparrow27) [reply zoo/libs/utils/filesystem.py]: About the one liner kinda functions . the excuse of "i cant remember how to do it so i'm gonna create a function" isn't a good reason to pollute the code base, slow down code execution and add a layer of abstraction."one liner for a one liner" make no sense.

Comment (andrewsilke) [reply zoo/libs/utils/filesystem.py]: got it good to go, all fixed

**Pull request merged**